reviews/2022-open-cluster-management-annual.md

```diff
@@ -0,0 +1,99 @@
+
+
+### Alignment with CNCF
+
+- OCM was accepted as a CNCF Sandbox project on November 09, 2021.
+
+## Development metrics
+
+The OCM devstats page and dashboards can be found [here](https://openclustermanagement.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m&search=open).
+
+- According to devstats, OCM currently has [64](https://openclustermanagement.devstats.cncf.io/d/22/prs-authors-table?orgId=1) contributors from [19](https://openclustermanagement.devstats.cncf.io/d/5/companies-table?orgId=1)
+companies. On average, there are [~1085 contributions per month](https://openclustermanagement.devstats.cncf.io/d/74/contributions-chart?orgId=1&var-period=m&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-company_name=All&var-company=all&from=now-1y&to=now) contained within [134 merged PRs per month](https://openclustermanagement.devstats.cncf.io/d/74/contributions-chart?orgId=1&var-period=m&var-metric=prs&var-repogroup_name=All&var-country_name=All&var-company_name=All&var-company=all&from=now-1y&to=now).
+- [New PRs in last year](https://openclustermanagement.devstats.cncf.io/d/15/new-prs-in-repository-groups?orgId=1).
+- The community has grown since the project entered the CNCF sandbox.
+  - We consistently hold weekly community meetings. The meeting records can be found [here](https://www.youtube.com/@OpenClusterManagement/videos) for different timezone/language community members. The average number of meeting attendees is ~10.
+  - Number of contributors: 120+ -> **150+**
+  - Github stars: 600+ -> **1000+**
+  - Github forks: 330+ -> **560+**
+  - Contributing organizations: 5 -> **19**
+
+Regarding these metrics, we would perceive the community as healthy and rising.
+
+## Maintainers
+
+We have established [the contributor ladder](https://github.com/open-cluster-management-io/community/blob/main/CONTRIBUTOR_LADDER.md), and quite a few active and qualified contributors joined to become members. We currently have [7 maintainers from 2 organizations](https://github.com/cncf/foundation/blob/main/project-maintainers.csv) as maintainers.
+
+- Qiu Jian (Red Hat)	@qiujian16
+- Joshua Packer	(Red Hat)	@jnpacker	
+- David Eads	(Red Hat)	@deads2k	
+- Mike Ng	(Red Hat)	@mikeshng	
+- Min Kim	(Alibaba)	@yue9944882	
+- Yong Feng	(Alibaba)	@luckyfengyong
+
+## Project adoption
+
+OCM has 
+
+
+## Project goals
+
+The primary goal of OCM is to enable capabilities within the Kubernetes ecosystem to orchestrate across multiple clusters and cloud providers. In the past year, we have achieved the following development outcomes:
+- Release cadence: 4 minor releases, roughly once every three months. The latest release is v0.9.0.
+- Key features added to the project (complete changelog can be found [here](https://open-cluster-management.io/community/releases/)):
+  - 0.6
+    - Cluster Proxy add-on.
```

Comment:
Maybe more meaningful:
- New add-ons
  - Cluster Proxy, allows a proxied connection from the hub to a managed cluster's api ingress
  - Managed Service account, allows the hub to define service account and receive its token on a managed cluster